etc/init.d/zfs.in

```diff
@@ -0,0 +1,74 @@
+# ZoL userland configuration.
+
+# Run `zfs mount -a` during system start?
+ZFS_MOUNT='yes'
+
+# Run `zfs unmount -a` during system stop?
+ZFS_UNMOUNT='yes'
+
+# Run `zfs share -a` during system start?
+# nb: The shareiscsi, sharenfs, and sharesmb dataset properties.
+ZFS_SHARE='yes'
```

Comment:
@FransUrbo Calm down. My main concern is a silent change in behavior that involves sharing data over the network. I'm advocating that this not be done silently, like the current situation in the zfs dailies that you're currently providing. They should tell the sysadmin (via NEWS.Debian for example) that this behavior has changed. That's just common courtesy.

And as for "Whether or not sharing should be on by default is it's own discussion.", I meant exactly that. I don't care strongly on that topic, but it is truly it's own discussion. And the Debian and RHEL: camps take very different views on this. IIRC, RHEL tends to not start sharing services/shares by default while Debian tends to.

In the future, please stop using straw man arguments. I never stated that my situation should be applied to all users. It makes it more difficult to have a productive conversation with you.

---Alex